test/test.js

```diff
@@ -15,3 +17,27 @@ test('Basic test', (t) => {
     t.end()
   })
 })
+
+test('Cache directory is moved to new location', (t) => {
+  // Download to old cache directory location
```

Comment:
:thinking: It's only temporary code though just for migrating, after some time it will be removed... Already spent a lot of time on it, apart from cleaning up the file it does the same as the steps you proposed right?

Thoughts on how important this is? Not saying I don't want to do it at all but I'm just considering the time it costs versus value that comes out of it.